packages/block-editor/src/content.scss

```diff
@@ -1,6 +1,7 @@
 @import "./components/block-icon/content.scss";
 @import "./components/block-list/content.scss";
 @import "./components/block-list-appender/content.scss";
+@import "./components/block-manager/style.scss";
```

Comment:
Should this rather be imported in `packages/block-editor/src/style.scss`?

Here's what I'm seeing

| In content.scss | In styles.scss |
|--------|--------|
| <img width="300" alt="Screenshot 2024-11-25 at 7 24 05 am" src="https://github.com/user-attachments/assets/2c14af54-39de-464c-af02-6f4c01c9fd62"> | <img width="300" alt="Screenshot 2024-11-25 at 7 39 48 am" src="https://github.com/user-attachments/assets/2afa997a-741c-490f-9e92-ec60164e14f4"> |

Reply:
Thanks for the review!

Strangely, I was not able to reproduce the issue. In my environment, `content.css` is loaded in the document root and the styles are loaded correctly:

![content](https://github.com/user-attachments/assets/94b47ea4-de15-4965-99d9-50d6ad53b7c2)


However, the styles should not be part of `content.css`, so I fixed it by 793d742487898b43a8a48876b6cf2fbd3d40c852.

Reply:
Oh, that is strange. I tried rebuilding when I tested to see if it was my local env just in case. Maybe it was 🤷🏻 

Anyway thanks for figuring it out!

Reply:
If your instance is not iframed, "content" will load in the root document, otherwise, it loads in the iframe I think.

Reply:
"style" seems like the right place for this style 👍